### ISSUE TITLE
Reusing Client causes gRPC Error

### ISSUE THREAD

**Issue by moritzploss-k** [goodhamgupta/authzed_ex#10]
Thanks for making this library!

I'm currently debugging an issue where I can first make a successful call to SpiceDB, but then get a gRPC error `:down :normal` on the second request, and `:down: :noproc` on all following requests. I'm re-using the same `client` object for all calls:

```elixir
alias Authzed.Api.V1

def call_spice_db(client) do
  client.permissions_service.write_relationships(client.channel, ...) 
end

def main do
  client = V1.Client.new("url:port", V1.GRPCUtil.insecure_bearer_auth_token("token"))
  {:ok, %{written_at: _}}                              = call_spice_db(client)
  %GRPC.RPCError{status: 2, message: ":down: :normal"} = call_spice_db(client)
  %GRPC.RPCError{status: 2, message: ":down: :noproc"} = call_spice_db(client)
end
```

I'm not a gRPC expert, but my understanding is that it should be possible to re-use channels for multiple requests, and therefore re-use the same `client` object. Any help would be much appreciated!

**Comment by goodhamgupta** [comment]
Hi @moritzploss-k,

Thanks for reporting the issue! I tried to replicate this error, but was unable to see the error you're facing. I executed the test available at in the [repo](https://github.com/goodhamgupta/authzed_ex/blob/main/test/authzed_test.exs#L46), where I
- Use the SpiceDB docker container
- Write multiple users and permissions using the same client.

Would it be possible to provide a detailed example which can replicate your error?
